pytest.ini

```diff
@@ -0,0 +1,13 @@
+# pytest.ini
+[pytest]
+minversion = 6.0
+addopts = -ra -q
+norecursedirs = .svn _build ndarray* # ignore tests in ndarray due to missing stdVector module not found
```

Comment:
All tests need to be ignored in ndarray?

Reply:
NumpyNdArray2StdVectorNdArray.py will completely fail. Some tests in AbstractNdArray.py will pass. I think this folder is testing converting between numpy nd array to c++ stdVectors. Looking into multiphone and mcvine folders ,the AbstractndArray.py is no longer used along with stdVector. So my recommendation is to not test these two. There is a separate story created to look into this whole NdArray and stdVector thing that Pete said he'll take a look down the road.